pkg/config/messages.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"time"
+
+	cerrors "github.com/pingcap/ticdc/pkg/errors"
+	"github.com/pingcap/ticdc/pkg/p2p"
+)
+
+// MessagesConfig configs MessageServer and MessageClient.
+type MessagesConfig struct {
+	ClientMaxBatchInterval TomlDuration `toml:"client-max-batch-interval" json:"client-max-batch-interval"`
+	ClientMaxBatchSize     int          `toml:"client-max-batch-size" json:"client-max-batch-size"`
+	ClientMaxBatchCount    int          `toml:"client-max-batch-count" json:"client-max-batch-count"`
+	ClientRetryRateLimit   float64      `toml:"client-retry-rate-limit" json:"client-retry-rate-limit"`
+
+	ServerMaxPendingMessageCount int          `toml:"server-max-pending-message-count" json:"server-max-pending-message-count"`
+	ServerAckInterval            TomlDuration `toml:"server-ack-interval" json:"server-ack-interval"`
+	ServerWorkerPoolSize         int          `toml:"server-worker-pool-size" json:"server-worker-pool-size"`
+}
+
+// read only
+var defaultMessageConfig = &MessagesConfig{
+	ClientMaxBatchInterval:       TomlDuration(time.Millisecond * 100),
+	ClientMaxBatchSize:           8 * 1024, // 8MB
+	ClientMaxBatchCount:          128,
+	ClientRetryRateLimit:         1.0, // Once per second
+	ServerMaxPendingMessageCount: 102400,
```

Comment:
Is `102400` a reasonable default value ?

Reply:
I think it is:
- MessageServer is a singleton.
- Individual messages are small. Large messages would be extremely rare.